R/cache.R

```diff
@@ -0,0 +1,54 @@
+ensure_cache = function() {
+  dir = tools::R_user_dir("oysteR", which = "cache")
+  path = file.path(dir, "cached-deps.rds")
```

Comment:
minor: Is there a way to declare this filename as a shared "constant"? The the string would not be duplicated here and in `remove_cache()`. It's already pretty safe, as it is encapsulated in `cache.R`, but thought I'd ask and learn. :)

Reply:
No constants in R. But I've created a single get method